test/skimNtuple_HHbtag.cpp

```diff
@@ -2459,11 +2441,6 @@ int main (int argc, char** argv)
 		}
 
 	  theSmallTree.m_pairType    = pType ;
-	  // Need to change the channel: LLR-> 0:muTau - 1:eTau  /  PI-> 0:eTau - 1:muTau
-	  if      (theSmallTree.m_pairType == 0) theSmallTree.m_BDT_channel = 1.;
```

Comment:
Can you please check if this m_BDT_channel is used somewhere else? I have the impression it unexpectedly was...

Reply:
I'm actually wondering whether we should add an option for ```mumu```. I've asked Elvira if mumu was taken into account.

Reply:
After doing a ```grep``` to the entire repository, I could not find any instance where we use that variable, except as an input to ```HH-bTag```.

Reply:
I was pretty sure HHBTag was the only place, thanks for checking :)

Reply:
I've added the MuMu channel to HH-bTag. This might hopefully explain the weird distributions we were seeing in the control regions.

Reply:
Update: HH-bTag was never trained for the MuMu channel, implying that we cannot use it. I am using DeepJet score for the moment, until a new training including MuMu is available.

Reply:
- I've removed the need to define similar vectors ```jets_and_BTag``` and ```jets_and_sortPar```: one is enough with minor modifications and is less error-prone
- I've rewritten ```bTagSF``` to avoid unnecessary arrays and separate ```for``` loops, thus increasing readability.

Reply:
I've added missing JER files for 2018, but  they seem to be missing in 2017 UL. Please follow [this thread](https://cms-talk.web.cern.ch/t/jers-for-ak8-in-2017/36614) for updates.

Reply:
Update: We should use AK4 JER files for both AK4 and AK8 jets in 2017.